common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -84,16 +97,23 @@ def test_can_create_and_edit_content_groups(self):
 
         self.assertIn("Updated Second Content Group", second_config.name)
 
-    def test_cannot_delete_content_group(self):
+    def test_can_delete_unused_content_group(self):
```

Comment:
I recommend leaving the original test but calling it ``test_cannot_delete_used_content_group``, and then also add this new test for the unused case.